yarn.lock

```diff
@@ -1226,6 +1226,14 @@
     core-js "^2.6.5"
     regenerator-runtime "^0.13.2"
 
+"@babel/runtime-corejs2@^7.6.3":
```

Comment:
Is there a reason these dependencies are updated in the lockfile? Or was this just yarn doing its thing to get all the latest things?

Reply:
Probably yarn doing its thing. This was a fresh clone.